ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/filter/FilterManager.java

```diff
@@ -151,28 +152,28 @@ public boolean uninstallFilter(final String filterId) {
     }
   }
 
-  public void recordBlockEvent(final BlockAddedEvent event, final Blockchain blockchain) {
+  public void recordBlockEvent(final BlockAddedEvent event, final Blockchain __) {
```

Comment:
I see that it is not being used by the method, but why the underscore variable name? Is it just how you are denoting that it is not used?

Reply:
Yup! Common functional pattern to let the reader know early that they don't have to worry about it.

Reply:
Should we handle that at the lambda level and for the method signature actually just remove the param?

Reply:
If we never use it then for sure, I'll have to look into that.

Reply:
You were right, it was pretty useless. It was only used in one place that already had access to the blockchain in the scope anyways!

Reply:
I made a separate PR removing it